flang/lib/Semantics/check-call.cpp

```diff
@@ -2020,6 +2020,20 @@ bool CheckPPCIntrinsic(const Symbol &generic, const Symbol &specific,
   return false;
 }
 
+bool CheckWindowsIntrinsic(
```

Comment:
This is a good idea. Are these the only two intrinsics which are not supported on Windows? There is at least ACCESS which is also unsupported.

Reply:
We should probably do a check through to see which intrinsics aren't working on Windows and add them all here. That should be a separate patch though

Reply:
Agreed. Please add a TODO here